charts/nexus3/templates/job-config.yaml

```
@@ -25,16 +25,10 @@ spec:
         - name: config
           image: {{ include "nexus3.image" .Values.config.job.image }}
           imagePullPolicy: {{ .Values.config.job.image.pullPolicy }}
+          {{- with .Values.nonRootPodsecurityContext }}
```

```suggestion
          {{- with .Values.config.job.securityContext }}
```
This is a security context, not a pod security context and I think it should be explicitly configured for the job. Could you also add a `config.job.podSecurityContext` value and implement it for the job pod?